apps/terminalclock/app.js

```diff
@@ -113,47 +114,57 @@
     g.drawString(line, 5, yPos, true);
   };
 
-  let drawTime = function(now, pos){
+  let drawTime = function(now, pos) {
     let h = now.getHours();
     let m = now.getMinutes();
     let time = ">" + (""+h).substr(-2) + ":" + ("0"+m).substr(-2);
     drawLine(time, pos);
   };
 
-  let drawDate = function(now, pos){
-    let dow = locale.dow(now, 1);
-    let date = locale.date(now, 1).substr(0,6) + locale.date(now, 1).substr(-2);
-    let locale_date = ">" + dow + " " + date;
-    drawLine(locale_date, pos);
+  let drawDate = function(now, isoDate, pos) {
```

Comment:
We don't use this parameter - let's delete it (and update call sites)

Reply:
Thanks, I think this must be left over from when I was trying to pass it before realising `clock` was in scope :+1:.